locksmith/src/websub/hub.ts

```diff
@@ -0,0 +1,63 @@
+import cron from 'node-cron'
+import fetch from 'cross-fetch'
+import { networks } from '@unlock-protocol/networks'
+import { Key } from '../graphql/datasource'
+import { Hook } from '../models'
+import { logger } from '../logger'
+
+logger.info('Locksmith hub started!')
+
+async function notifyHook(hook: Hook, body: any) {
+  const { callback } = hook
+  try {
+    const response = await fetch(callback, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(body),
+    })
+
+    if (response.status !== 200) {
+      throw new Error('Hook returned non-200 status code')
+    }
+    logger.info(`Hook ${hook.id} notified`)
+  } catch (error) {
+    logger.error(`Hook ${hook.id} failed to notify`)
+  }
+}
+
+cron.schedule('* * * * *', async () => {
+  const hooks = await Hook.findAll({
+    where: {
+      mode: 'subscribe',
+      topic: 'keys',
+    },
+  })
+
+  const subscribedHooks = hooks.filter((hook) => hook.mode === 'subscribe')
```

Comment:
Isn't that duplicate with the `where` in the query above?

Reply:
oh right. I will remove it.